app/components/review_component.rb

```diff
@@ -48,7 +48,7 @@ def default_classes
     end
 
     def edit_link
-      govuk_link_to text, href, aria: { label: "#{text} #{title}" } if text && href
+      govuk_link_to text, href, aria: { label: "#{text} #{title}" }, classes: "govuk-!-display-none-print" if text && href
```

Comment:
Why is this one here?

Reply:
Well I can see why it's here once I scrolled down lol, but why was this rule in place to begin with?